### ISSUE TITLE
Dump of UI/UX concerns and suggestions

### ISSUE THREAD

**Issue by locness3** [ideoxan/ideoxan#101]
Hi! Below is a dump of UI and UX things that concern me, and suggestions on how to improve, in no particular order.
For info, I'm testing on Firefox 82.0 on Ubuntu 20.04.1 LTS.

First of all, most text is too small! All text should be at least over 10px to be comfortable and legible. I'd use at least 13px.

Then, these icons are way too big, especially compared to the text. [You shouldn't scale up icons designed at small sizes](https://medium.com/refactoring-ui/7-practical-tips-for-cheating-at-design-40c736799886#e14f).
![Capture du 2020-10-25 09-53-58](https://user-images.githubusercontent.com/37651007/97102744-50274600-16a8-11eb-973f-60c41631acc1.png)

Then, the pricing table could be presented in a "friendlier" and easier sacannable way, using cards  and showing only the most important points, also emphasizing one of the two subscriptions more, for example.

Finally, there is a general lack of contrast in some places. For example, the primary button text color has a contrast of only 3.60 from the background, and the IX+ yellow has a 1.30 contrast with the page background (in light mode). The WCAG recommended value is of at least 4.5 for regular text and 3 for large text.

**Comment by Skyclo** [comment]
Okay so I read this before and I do hear your point. I'll try to address each one but this might not make sense because this is just me listing different points

UI on this website has been a point of concern for me for a while. The problem is that I rushed the development of the main website. I based everything around the editor, which to be honest, had a really bad interface in the first place. The CSS, let alone the HTML, of the website is barely readable or easy to modify (ArduinoidIOT can attest to that). Overall, it is consistent and is in need of a refresh. However that refresh is going to take up more development time in a project that is already months behind schedule for its release. Doing a refresh of the UI is something that I looked at months prior and saved for post-launch. But as I have gone on with development, the UI has grown bulkier and harder to manage. A UI overhaul is inevitable at this point. I looked into different frameworks and decided to settle with TailwindCSS. It seems like the best library to get the job done and has minimal impact on the design process. This would help clean up the site, ensure uniformity, and tackle most (if not all) of the issues presented.

The font on the website is 12px. I can up it to 13px if needed but that may conflict with spacing/display in the editor. I am planning to replace the icons with something from https://undraw.co/ today. When it comes to pricing, that issue will be discussed later in a separate issue. Do you have any suggestions regarding the text contrast?

By the way that article seems good. I'll try and implement some of the suggestions.

**tl;dr:** I hear you out on the UI. Overhaul coming soon. Working on temporary UI changes to fix some of the issues.

**Comment by Skyclo** [comment]
Hey so yesterday I pushed some commits that cleaned up the homepage UI a bit. Let me know what you think.

**Comment by locness3** [comment]
I'll take a look, if I remember.

**Comment by Skyclo** [comment]
I uploaded screenshots of them. Sorry if they look a bit funny.... they were taken on a remote computer. 

![image](https://user-images.githubusercontent.com/39171349/98585167-dc369180-2294-11eb-9e19-99071dbe3b95.png)
![image](https://user-images.githubusercontent.com/39171349/98585259-fa9c8d00-2294-11eb-94d9-45fdf682dbd6.png)

**Comment by locness3** [comment]
Better, athough images could be smaller or text could have more line-height

**Comment by micahlt** [comment]
@Skyclo I think you should also add some more padding on the left and right sides.  It looks a little odd on widescreen devices.  People also prefer to read more shorter lines of text than a few long lines, so maybe reduce the width of the textboxes?

**Comment by Skyclo** [comment]
The website is currently being redone using tailwindcss. This should make the UI a lot better (as you can tell css is not my specialty lol)
